src/jdmtool/featunlk.py

```diff
@@ -387,11 +388,20 @@ def display_content_of_dat_file(dat_file: pathlib.Path):
                 header_bytes = fd.read(0x200)
                 fd.seek(-0x102, os.SEEK_END)
                 footer_bytes = fd.read(0x102)
```

Comment:
Need to initialize `footer2_bytes` here, too, just in case (even if it's unused). Otherwise, tools like e.g. pylance show a warning.